From b2ebb64c1d6797f7b9e8bda0cbbbe61484fb54a2 Mon Sep 17 00:00:00 2001
From: Martin Pitt <martin.pitt@ubuntu.com>
Date: Mon, 5 Sep 2011 15:29:20 +0200
Subject: [PATCH 16/18] Define "FolderSeparator" and "LineEnding" properties

Some players require DOS folder separators in playlists. In the future, some
might also require different line separators, so define two new properties for
these.

https://bugs.freedesktop.org/show_bug.cgi?id=40441
---
 README |   10 ++++++++--
 1 files changed, 8 insertions(+), 2 deletions(-)

diff --git a/README b/README
index 4ac98b6..c3d9333 100644
--- a/README
+++ b/README
@@ -73,9 +73,15 @@ The [Playlist] section describes the player playlist capabilites. If it's
 absent, it means the player doesn't support playlists (or that the information
 contained in media-player-info is incomplete, in such a case, please email us
 or file a bug).
-- Formats: it's a semi-colon-separated list of the mime-types of the
+- Formats: a semi-colon-separated list of the mime-types of the
   playlist formats supported by the device
-
+- FolderSeparator: separator between directory/file names in playlist files. If
+  not given, '/' is assumed (standard Unix separator). Some players require the
+  DOS separator '\'.
+- LineEnding: line separator in playlist files. This can have the values 'CR',
+  'LF', or 'CRLF', which correspond to 0x0D (\r), 0x0A (\n), and 0x0D0A (\r\n)
+  respectively. If not given, the default behaviour depends on the music player
+  software.
 
 The [storage] section describes the location where media files can be found and
 have to be stored.
-- 
1.7.6

